exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,21 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  return num < 0;
+}
+
+function isBetween5and10(num) {
+  return 5 <= num <= 10;
+}
+
+function isShortName(name) {
+  return name.length < 10;
+}
+
```

Comment:
Nice. In Javascript, (and most other languages), the first element in an array has an index of 0.